.build/dependency-check-suppressions.xml

```diff
@@ -49,12 +49,6 @@
         <packageUrl regex="true">^pkg:maven/net\.openhft/chronicle\-wire@.*$</packageUrl>
         <cpe>cpe:/a:wire:wire</cpe>
     </suppress>
-    <suppress>
```

Comment:
Nice catch

Reply:
Yes, this one was an interesting case. Google engineers first considered it as not a meaningful thing to fix. They just deprecated it and asked people to use Java.nio if they need a more secure way. So I was considering a checkstyle rule for the Cassandra codebase (some time ago, this was suppressed, and we used the method in testing). But then it seems too many people complained, and eventually, the team fixed the method, and here we are. :-)